addons/payment/data/payment_method_data.xml

```diff
@@ -816,7 +816,7 @@
 
     <record id="payment_method_cash_app_pay" model="payment.method">
         <field name="name">Cash App Pay</field>
-        <field name="code">cash_app_pay</field>
+        <field name="code">cashapp</field>
```

Comment:
You can't simply change pm_code of payment method instead you can add this `PAYMENT_METHODS_MAPPING` variable of stripe and ayden and may be update this in master with upgrade script

Reply:
Done the changes 
thankyou